features/email.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Import and Export Email Groups
+
+    Scenario: Importing and exporting a field group with an email field
+      Given a WP install
+      When I run the wp-cli command to import the file "email-group.json"
```

Comment:
Like mentioned before, is it possible to show the real command instead of a description?